DO-02

FedorFedichkinTR opened the pull request:
Project structure was changed
ConnectionPool was refactored
H2 database was connected
Logger log4j was implemented

FedorFedichkinTR committed:
Project structure was changed
ConnectionPool was refactored
H2 database was connected
Logger log4j was implemented

philkuzmin commented on src/main/webapp/static/js/bootstrap.js:
```diff
@@ -1538,7 +1538,7 @@ var Dropdown = function ($) {
 
       // if this is a touch-enabled device we add extra
-      // empty mouseover listeners to the body's immediate children;
+      // empty mouseover com.epam.listeners to the body's immediate children;
```
с автопереименованием нужно быть очень осторожным:) комменты - ерунда, но он может случайно переименовать кучу всего важного.

FedorFedichkinTR replied:
упс..
да, надо будет проверять в будущем

FedorFedichkinTR committed:
Project structure was changed
ConnectionPool was refactored
H2 database was connected
Logger log4j was implemented

FedorFedichkinTR committed:
Merge branch 'dev' into DO-02

# Conflicts:
#	src/main/java/com/epam/dao/AbstractDaoFactory.java
#	src/main/java/com/epam/dao/UserDao.java
#	src/main/java/com/epam/dao/h2/H2DaoFactory.java
#	src/main/java/com/epam/listeners/DataBaseInitListener.java
#	src/main/java/com/epam/model/Roles.java
#	src/main/java/com/epam/model/RoomType.java
#	src/main/java/model/Roles.java
#	src/main/java/model/RoomType.java
#	src/main/java/model/enums/Roles.java
#	src/main/java/model/enums/RoomType.java
